dcicutils/ecr_utils.py

```diff
@@ -38,13 +42,19 @@ def resolve_repository_uri(self, url=None):
             try:
                 resp = self.client.describe_repositories()
                 for repo in resp.get('repositories', []):
-                    if self.env in repo['repositoryUri']:
-                        return repo['repositoryUri']
+                    if repo['repositoryUri'].endswith(self.env):
```

Comment:
Less worried about this failing, though theoretically `.endswith('/' + self.env)` is a little better. This is good enough.